infra/upload_images/container_ocr/handler/ocr_processor.py

```
@@ -199,6 +203,295 @@ def _process_refinement_job(
+
+        region = {
+            "x": float(ocr_job.reocr_region.get("x", 0.70)),
+            "y": float(ocr_job.reocr_region.get("y", 0.0)),
+            "width": float(ocr_job.reocr_region.get("width", 0.30)),
+            "height": float(ocr_job.reocr_region.get("height", 1.0)),
+        }
+
+        ocr_json_path = download_file_from_s3(
+            ocr_routing_decision.s3_bucket,
+            ocr_routing_decision.s3_key,
+            Path("/tmp"),
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)